### DISCUSSION TITLE
MacOS Silicon .app plz plz?

### DISCUSSION THREAD

**Discussion by KamillaPup** [dmMaze/BallonsTranslator#599] [General]
I'm having a hell of a time trying to build this app, and it would be so awesome if someone else with an M-series MacOS box to build one that could be shared.

**Comment by dmMaze** [comment]
Just run the source code directly. There's no actual benefit that comes out of the build except saving you some clicks anyway.

**Reply by kemopup** [reply]
Running from source gives me errors too on MacOS. It works fine on my Windows desktop without issue. I tried in my Windows VM on my silicon Mac, and it complained about no Nvidia driver.

**Reply by dmMaze** [reply]
> Running from source gives me errors too on MacOS. It works fine on my Windows desktop without issue. I tried in my Windows VM on my silicon Mac, and it complained about no Nvidia driver.

So what errors does it spit when running it directly? (not inside VM)
